data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,436 @@
+# Copyright (c) 2020 MasterOfGrey
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+########################################################################
+
+event "secret mountaintop"
+	# Ensure Mountaintop exists so the next mission can trigger
+	system "Ultima Thule"
+		government "Uninhabited"
+		object
+			sprite star/f0
+			period 10
+		object
+			sprite planet/desert5
+			distance 200
+			period 22.3607
+		object
+			sprite planet/europa-b
+			distance 403.41
+			period 64.056
+		object
+			sprite planet/desert7-b
+			distance 824.9
+			period 187.302
+		object
+			sprite planet/gas1
+			distance 1706.99
+			period 557.553
+			object
+				sprite planet/ice8-b
+				distance 216
+				period 11.1964
+		object "Wormhole Alpha"
+			sprite planet/wormhole
+			distance 4208.63
+			period 2158.5
+			object Mountaintop
+				sprite "effect/flotsam yottrite"
+				distance 315
+				period 31.1964
+	planet Mountaintop
+		government "Unknown"
+		# Placeholder description; you can't actually land.
+		description `This tiny asteroid has a space hazard warning transmitter, of a type typically used to mark construction sites. Perhaps someone intends to build a space station here?`
+		"required reputation" 1000
+		bribe 0
+
+mission "Hai Building Mountaintop [1] Construction Fleet"
+	name "Escort construction fleet to Ultima Thule"
+	description "Meet a construction fleet at <stopovers> and escort them to to <destination> to prevent a confrontation."
+	autosave
+	landing
+	source "Hai-home"
+	stopover "Makerplace"
+	destination "Mountaintop"
+	clearance
+	to offer
+		has "Hai Leaks Response 6: done"
+	on offer
+		conversation
+			`When you return to Hai-home you are directed now to a government hangar where you are met by Xilin and Sayari, both looking very relieved.`
```

Comment:
These commas make it feel stilted to me, you sure?

Reply:
Just double checked both of them. The first is definitely necessary, but the second is up for debate since it's hard to determine whether it's a restrictive clause, so I think taking it out wouldn't be a huge deal.
```suggestion
			`When you return to Hai-home, you are directed to a government hangar where you are met by Xilin and Sayari, both looking very relieved.`
```
```